python/README.md

```diff
@@ -0,0 +1,13 @@
+# Kenna Security Python Blog Samples
+
+Contains Kenna Security code samples that are relevant to Kenna Security blogs. The directories
+reflect the sections in the [API documentation](https://apidocs.kennasecurity.com/reference).
+
+The examples use the environment variable `API_KEY`. Please make sure you follow the
+[authentication directions](https://apidocs.kennasecurity.com/reference#authentication) 
+ and export the API_KEY in your shell environment.
+
+The python samples use the [requests library](https://requests.readthedocs.io/en/master/) to communincate
+to Kenna Security servers over HTTP.
+
+These samples were tested using Python 3.8.7.
```

Comment:
Can we slap a `.python-version` file in the directory to be explicit?

Reply:
Done.  What is your opinion about having it .gitignore?

Reply:
Since @sfgeorge hasn't responded, I'm going to not add .python-version to .gitignore.  It can be done later.

Reply:
I'm open to your thoughts on this one.  And if you'd lean towards git-ignoring `.python-version` I can get behind that.

`.python-version` obviously should not be checked-in library, such as an SDK.  But for an application (which this feels more like to me), it's okay to be explicit about a python/language version.

I use the same distinction for ...`.lock` files...  In a library: no, In an application: possibly yes.